src/material/tabs/tabs-module.ts

```diff
@@ -62,6 +62,7 @@ import {MatPaginatedTabHeader} from './paginated-tab-header';
     _MatTabNavBase as any,
     _MatTabBodyBase as any,
     _MatTabHeaderBase as any,
+    _MatTabLinkBase as any,
```

Comment:
I think we only need `any` if the class is actually `abstract`. But it's not a big deal here.